src/components/DateTimePicker/__test__/pickerModal.spec.js

```diff
@@ -17,7 +17,7 @@ describe('<DateTimePickerModal/>', () => {
     });
     it('should pass the right time to TimeSelect component', () => {
         const component = mount(<DateTimePickerModal isOpen value={value} />);
-        expect(component.find('TimeSelect').props().value).toBe('08:48 PM');
+        expect(component.find('TimeSelect').props().value).toBe('12:00 AM');
```

Comment:
since we are passing as value:
```const value = new Date('06/01/2019 20:48:34');```
the time passed to TimeSelect should be 08:48 PM